scripts/test.sh

```diff
@@ -365,21 +369,22 @@ case "$TARGET" in
   stop_server
   ;;
 "electron")
-  npm install --no-save
+  npm ci
```

Comment:
Awesome! I didn't know that command :)

Reply:
It's a fairly recent addition that makes it just install the versions from package-lock rather than rerunning dependency resolution. Seems to knock a few seconds off installation for us.